doc/Getting-started/index.md

```diff
@@ -65,543 +56,204 @@ To use Kaa Sandbox, your system must meet the following minimum system requireme
+      | TCP      | 9888 | 0.0.0.0/0  |
+      | TCP      | 9887 | 0.0.0.0/0  |
+      | TCP      | 9080 | 0.0.0.0/0  |
+   
+   * Select the created security group.
 
-## Create notification schema
+7. On **Review Instance Launch** step, make any changes as appropriate.
 
-The application that you have created in the previous step already includes the default versions of the profile, configuration, notification and log schemas ready for use. However, in this sample application, we will use a custom notification schema for demonstration purposes. To create and upload the schema, proceed as follows:
+8. Click **Launch** and then in the **Select an existing key pair** or **Create a new key pair** dialog, do one of the following:
 
-1. Create the _schema.json_ file on your PC with the following schema definition:
+   * Select an existing key pair from the **Select a key pair** drop list.
 
-   ```json
-   {
-     "type": "record",
-     "name": "Notification",
-     "namespace": "org.kaaproject.kaa.schema.example",
-     "fields":[
-        {
-          "name": "message",
-          "type": "string"
-        }
-      ]
-    }
-```
+   * If you need to create a new key pair, click **Create a new key pair**. Then create the new key pair as described in [Creating a key pair](http://docs.aws.amazon.com/gettingstarted/latest/wah/getting-started-prereq.html).
 
-2. Open the admin UI in your browser ( [http://127.0.0.1:8080](http://127.0.0.1:8080/) ) and log in as a tenant developer (user/password: devuser/devuser123).
-3. Open the relevant **Notification schemas** window (**Applications =\> My First Kaa Application =\> Schemas =\> Notification**) and click **Add new schema**.
-4. In the **Add new schema** window, fill in the fields as shown in the following screenshot and then click **Add**.   
-<img src="attach/image2015-6-15 20-9-14.png" width="800" height="400">
+9. Click **Launch Instances**. The **Launch Status** page will be displayed.
 
-5. Scroll down and use the Upload from file function to find the previously created json file with the schema. Alternatively, you can use the Schema [Avro UI form***]() to create the schema.
-6. Click Upload.  
-<img src="attach/image2015-6-19 14-51-26.png" width="900" height="500">
-7. Click Add at the top of the window.
+10. Click **View Instances**.
 
-As a result of this operation you will see two notification schemas in the list:
-<img src="attach/image2015-6-15 20-18-54.png" width="1200" height="300">
+11. After launching Kaa Sandbox instance, go to **<your\_instance\_public\_dns\>:9080/sandbox** or **<your\_instance\_public\_ip\>:9080/sandbox** URL. Public DNS or IP of your instance are available from your instance description.
 
-In this screenshot, version 2.0 is the notification schema that was just created. We will use this version for the SDK generation in the next step.
+<br>
 
-## Generate SDK
+</div></div>
 
-To generate the SDK for the new application, proceed as follows:
+### Known issues
```

Comment:
Usually _Known issues_ falls into the category of bugs that development team aware about. Consider merging it with _Troubleshooting_ section.

Also it is better to move _Troubleshooting_ at the bottom of the page.